### PR TITLE
Expect UAE when running inside an appcontainer

### PR DESCRIPTION
**Pull request by ViktorHofer**
Fixes https://github.com/dotnet/corefx/issues/17934

### CODE REVIEW

File: src/System.IO.FileSystem.DriveInfo/tests/DriveInfo.Windows.Tests.cs [hunk 1]
```diff
@@ -154,22 +154,35 @@ public void TestInvalidDiskProperties()
         [PlatformSpecific(TestPlatforms.Windows)]
         public void GetVolumeLabel_Returns_CorrectLabel()
         {
-            int serialNumber, maxFileNameLen, fileSystemFlags;
-            int volNameLen = 50;
-            int fileNameLen = 50;
-            StringBuilder volumeName = new StringBuilder(volNameLen);
-            StringBuilder fileSystemName = new StringBuilder(fileNameLen);
+            void doDriveCheck()
```
Comment (stephentoub) [inline src/System.IO.FileSystem.DriveInfo/tests/DriveInfo.Windows.Tests.cs]: Nit: do => Do
(though to be fair it is a "local" function and we say locals should be camel cased, but I think local functions should be named more like members than like locals)

Comment (ViktorHofer) [inline src/System.IO.FileSystem.DriveInfo/tests/DriveInfo.Windows.Tests.cs]: I don't have a strong opinion on either. Do we have strict coding guidelines for that?

Comment (stephentoub) [inline src/System.IO.FileSystem.DriveInfo/tests/DriveInfo.Windows.Tests.cs]: We should.

**Comment by danmosemsft** [comment]
@dotnet-bot test innerloop portablelinux debug x64 build and test (https://github.com/dotnet/corefx/issues/19236)

**Pull request merged**